client/components/section-header/style.scss

```diff
@@ -35,11 +35,15 @@
 
 .section-header__label {
 	color: $gray;
+	font-size: 14px;
+}
+
+.section-header.header-is-compact .section-header__label {
```

Comment:
Do we ever apply the `header-is-compact` class? I did a search for that string using Sublime, and only found this one instance above.